agent/exploits/webexploit.py

```diff
@@ -7,16 +7,25 @@
 from requests import exceptions as requests_exceptions
 
 from agent import definitions
+from packaging import version
 
 MAX_REDIRECTS = 2
 DEFAULT_TIMEOUT = 90
 
+# This is the version pattern list
+# Please Add your version pattern if none of them match it
+# and make sure to put an example to clarify
+version_pattern = [
+    re.compile(r"\bv\d+\.\d+\.\d+(?:-\d+)?\b")
+]  # e.g., "v5.2.19-3" or "v1.2.3-4"
+
```

Comment:
I don't get what is the use of this?

Reply:
as versions can have multiple forms, whenever someone want to add a detection that's based on version he will add his version pattern to the list if it's not already there.